modules/holiday-snow.php

```diff
@@ -108,6 +108,19 @@ function jetpack_holiday_snow_option_name() {
 	return apply_filters( 'jetpack_holiday_snow_option_name', 'jetpack_holiday_snow_enabled' );
 }
 
+function jetpack_show_holiday_snow_option() {
+	// Always show snow option if a custom snow season has been set.
+	if ( has_filter( 'jetpack_is_holiday_snow_season' ) ) {
+		return true;
+	}
+
+	$today            = time();
+	$first_option_day = mktime( 0, 0, 0, 11, 24 ); // Nov 24
+	$last_option_day  = mktime( 0, 0, 0, 1, 11 );  // Jan 11
```

Comment:
I don't know if I would leave the option available after January 4th; wouldn't it be confusing to see the option, but not be able to use the feature?